agent/agentbootstrap/bootstrap_test.go

```diff
@@ -204,9 +204,6 @@ func (s *bootstrapSuite) TestInitializeState(c *gc.C) {
 	modelTag := model.Tag().(names.ModelTag)
 	controllerTag := names.NewControllerTag(controllerCfg.ControllerUUID())
 	s.assertCanLogInAsAdmin(c, modelTag, controllerTag, testing.DefaultMongoPassword)
-	user, err := st.User(model.Owner())
-	c.Assert(err, jc.ErrorIsNil)
-	c.Check(user.PasswordValid(testing.DefaultMongoPassword), jc.IsTrue)
```

Comment:
We need to keep this in some way, using the new infra. i.e. was the model owner created, if so, did they have the correct password?

I'm fine if you want to add it in another PR, it just needs doing.

Reply:
Ah yea, that makes sense. I had thought this was being tested at the wrong level perhaps. I'm not sure how to get ahold of the service factory from this point. A pointer to the state is returned from the function but there is no obvious way I can see to get the service factory. Should I add a pointer to also return that?

Reply:
Having discussed this in person, we can just fallback on the bash integration tests here. We have tests that cover accessing the API server and tests changing the admin password.